db/migrate/20170207192912_encrypt_otp_secret.rb

```diff
@@ -0,0 +1,22 @@
+class EncryptOtpSecret < ActiveRecord::Migration
+  class User < ActiveRecord::Base
+  end
+
+  def up
+    encrypt_otp_secret_key
+  end
+
+  def down
+  end
+
+  def encrypt_otp_secret_key
+    User.where.not(otp_secret_key: nil).find_in_batches.with_index do |users, batch|
+      users.each do |user|
+        encrypted_attribute = EncryptedAttribute.new_from_decrypted(user.otp_secret_key)
```

Comment:
How would we ensure the script will get run as part of the deployment?

Peter and I tested this migration locally and it ran fine. The only thing this migration relies on is the presence of the `EncryptedAttribute` class and the `user` responding to `otp_secret_key`, which will be true everywhere we deploy.